server/src/promo-dialog.js

```diff
@@ -0,0 +1,51 @@
+const React = require("react");
+const PropTypes = require("prop-types");
+const { Localized } = require("fluent-react/compat");
+const sendEvent = require("./browser-send-event.js");
+
+exports.PromoDialog = class PromoDialog extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if (this.props.display) {
+      this.props.promoOpen(this.props.promoType);
+      return <div id="promo-dialog-panel" className="promo-panel" >
+        <div className={`${this.props.promoType} promo-box default-color-scheme`}>
+        <a className="box-close" onClick={this.closePanel.bind(this)}></a>
```

Comment:
This should probably be a button (though I'm not sure if that matters much), and should have an aria-title or something on it. We don't currently have a string for just "close", but as I think about it, the string should be less ambiguous anyway, like "close notification".

Reply:
Updated to and use aria-label as "Close Notification"